app/src/main/res/layout/activity_option2.xml

```diff
@@ -42,5 +42,23 @@ android:padding="@dimen/large_spacing">
     android:text="@string/tv_output_init"
     android:textAppearance="@style/text_title"
     android:background="@color/colorAccent"/>
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+        <Button
+            android:id="@+id/back"
+            android:layout_width="173dp"
+            android:layout_height="wrap_content"
+            android:text="@string/back"
+            android:onClick="onClickBack" />
```

Comment:
Do we still need this? (don't have to create a new pull request)

Reply:
sorry , I forgot to delete these.